src/atc/orchestrators/eh2bronze/EhToDeltaBronzeTransformer.py

```diff
@@ -0,0 +1,73 @@
+import pyspark.sql.functions as f
+from atc_tools.time import dt_utc
+from pyspark.sql import DataFrame
+
+from atc.etl import Transformer
+from atc.tables import TableHandle
+
+
+class EhToDeltaBronzeTransformer(Transformer):
+    """
+    This class transforms eventhub data into the following schema:
+
+    | BodyId | Body | EnqueuedTimestamp | StreamingTime | pdate |
```

Comment:
you have to keep the properties column also. We have seen cases where e.g. the message type was added exclusively in the properties column

Reply:
I have now included the full eventhub schema